{
  "class": "vulnerability",
  "id": "oval:com.ubuntu.trusty:def:2021320520000000",
  "version": "1",
  "metadata": {
    "Title": "CVE-2021-32052 on Ubuntu 14.04 LTS (trusty) - medium.",
    "Description": "In Django 2.2 before 2.2.22, 3.1 before 3.1.10, and 3.2 before 3.2.2 (with Python 3.9.5+), URLValidator does not prohibit newlines and tabs (unless the URLField form field is used). If an application uses values with newlines in an HTTP response, header injection can occur. Django itself is unaffected because HttpResponse prohibits newlines in HTTP headers.",
    "affected": {
      "family": "unix",
      "Platform": "Ubuntu 14.04 LTS"
    },
    "reference": {
      "source": "CVE",
      "ref_id": "CVE-2021-32052",
      "ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-32052"
    },
    "advisory": {
      "Severity": "Medium",
      "Rights": "Copyright (C) 2021 Canonical Ltd.",
      "PublicDate": "2021-05-06 16:15:00 UTC",
      "Bug": [
        "http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=988136"
      ],
      "cve": {
        "text": "CVE-2021-32052",
        "href": "https://ubuntu.com/security/CVE-2021-32052",
        "public": "20210506",
        "cvss_score": "6.1",
        "cvss_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N",
        "usns": "4975-1,5373-1,5373-2"
      },
      "PublicDateAtUsn": "2021-05-06 16:15:00 UTC",
      "AssignedTo": "mdeslaur",
      "DiscoveredBy": "",
      "Crd": "",
      "Ref": null
    }
  },
  "notes": {
    "Note": "alexmurray> Requires Python 3.9.5 or greater which is only present in impish+ mdeslaur> Python 3.9.5 is now being backported to focal+, so this now needs to be fixed While this was originally marked as not-affected for bionic, a subsequent security update introduced the behaviour change in bionic which resulted in this CVE needing to be fixed."
  },
  "criteria": {
    "criterions": [
      {
        "test_ref": "oval:com.ubuntu.trusty:tst:2019123080000000",
        "comment": "python-django: while related to the CVE in some way, a decision has been made to ignore this issue (note: 'end of standard support')."
      }
    ]
  }
}
